static/js/searchDocs.js

```diff
@@ -0,0 +1,39 @@
+function doClick() {
+       if(String(document.location.href).includes("/dataingest/upload")){
```

Comment:
You are translating the href into a string each time you do a comparison. Just do it once and put it into a variable